create contact us form

saiharsha-22 opened the pull request:
solved the issue [CREATE] Contact Us #189

sujata13 commented:
Can you please also attach the snippet of the page that you have created. You can share it here in comment section or on our [Telegram Group](https://t.me/hacktober_fest_2020).

saiharsha-22 commented:
@sujata13 I have added in the telegram channel before I have added once again.
![WhatsApp Image 2020-10-03 at 8 41 55 PM](https://user-images.githubusercontent.com/61947484/94995027-125b6400-05b9-11eb-97ef-6915a38cf926.jpeg)

sujata13 commented:
Thanks for contribution. Happy Coding!!!

Ujaan07 commented on Day02/Contact us/contactus.html:
```diff
@@ -0,0 +1,124 @@
+           <p>+800 3005 4300</p>
+            <p class="blue">Contact Support</p>
+            <p>Our technical support is available by phone or email form 11am to 
+              11pm BST,Monady through Friday
```
Please change  monady to Monday.

Ujaan07 commented on Day02/Contact us/contactus.html:
```diff
@@ -0,0 +1,124 @@
+            <p>+800 3005 4300</p>
+             <p class="blue">Contact Sales</p>
+             <p>Our technical support is available by phone or email form 11am to 
+               11pm BST,Monady through Friday
```
Here also Monday instead of Monady
